docs/contributing-guide.adoc

```diff
@@ -0,0 +1,88 @@
+== AeroGear Service Developer Guide
+
+Following guide describes architecture for IOS SDK and instructions for developing new AeroGear Services.
+
+For instructions about using SDK in user applications please follow link:./getting-started.adoc[User Getting Started Guide]
+
+image:./images/diagram.svg[diagram,450,450,role="right"]
```

Comment:
This diagram file will need to be updated to reflect Android-y stuff. I can do this if needed in a separate PR.

Reply:
I could update it. Can you give me some example what will be the best to get that in?

Reply:
Probably something like [Auth] and [Metrics], replacing "Swift source code" and "Podspec file". Everything else seems accurate.